### DISCUSSION TITLE
issue:Change Detection

### DISCUSSION THREAD

**Discussion by frezeen** [tteck/Proxmox#2070] [Help Needed]
when stare a new page monitor and use:
Playwright Chromium/Javascript via 'ws://127.0.0.1:3000/?stealth=1&--disable-web-security=true'

i got this error:

https://changedetection.io/CHANGELOG.txt  Create a link to share watch config with others Using a chrome browser
WebSocket error: connect ECONNREFUSED 127.0.0.1:3000 =========================== logs =========================== <ws connecting> ws://127.0.0.1:3000/ <ws error> ws://127.0.0.1:3000/ error connect ECONNREFUSED 127.0.0.1:3000 <ws connect error> ws://127.0.0.1:3000/ connect ECONNREFUSED 127.0.0.1:3000 <ws disconnected> ws://127.0.0.1:3000/ code=1006 reason= ============================================================
changedetection.io

fresh LXC install

**Comment by frezeen** [comment]
can this help?
Collecting dnspython==2.2.1
  Downloading dnspython-2.2.1-py3-none-any.whl (269 kB)
     |████████████████████████████████| 269 kB 3.6 MB/s 
ERROR: changedetection-io 0.45.7.3 has requirement dnspython~=2.4, but you'll have dnspython 2.2.1 which is incompatible.
Installing collected packages: dnspython
  Attempting uninstall: dnspython
    Found existing installation: dnspython 2.4.2
    Uninstalling dnspython-2.4.2:
      Successfully uninstalled dnspython-2.4.2
Successfully installed dnspython-2.2.1

**Reply by tteck** [reply]
removed dnspython-2.2.1

**Reply by frezeen** [reply]
ok, no errors in install process now but playright continue with same error:
WebSocket error: connect ECONNREFUSED 127.0.0.1:3000 =========================== logs =========================== <ws connecting> ws://127.0.0.1:3000/ <ws error> ws://127.0.0.1:3000/ error connect ECONNREFUSED 127.0.0.1:3000 <ws connect error> ws://127.0.0.1:3000/ connect ECONNREFUSED 127.0.0.1:3000 <ws disconnected> ws://127.0.0.1:3000/ code=1006 reason= ============================================================

**Reply by gumbald** [reply]
Did you solve this? Mine works after a reboot but then stops a while later...

Error in my logs is about browserless failing with "com-kill".

**Reply by maxfun73v2** [reply]
Same here, the error comes back after a while, a reboot of the lxc container temporary solve the problem.
